guide/resources/code-resources.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: guide
+title: Code resources
+description: Bitcoin Design Guide code resources.
+nav_order: 4
+parent: Resources
+permalink: /guide/resources/code-resources/
+main_classes: -no-top-padding
+---
+
+<!--
+
+Editor's notes
+
+Illustration source:
+
+-->
+
+# Code resources
+
+Although mostly focused on design, some members of the community also have their hands in various code resources that are based on, or influenced by the Bitcoin Design Guide. They are all open source and free to use, modify and contribute to.
+
+## Bitcoin UI Kit packages
+
+The Bitcoin UI Kit is a great resource for designing bitcoin products. While the main effort is on the design components and Figma files, there are a couple of code implementations by community memebers that make adopting the designs in the guide much easier.
```

Comment:
```suggestion
The [Bitcoin UI Kit](https://www.bitcoinuikit.com) is a great resource for designing bitcoin products. While the main effort is on the design components and design source files, there are a couple of code implementations by community members that make adopting the designs in the guide much easier.
```
Adding a link for convenience, type, and removing Figma to be more general (ahem, Penpot, ahem).